### PR TITLE
Switch rusagelogfile to use options.Out

### PR DESCRIPTION
**Pull request by rhatdan**
When vendoring into Podman, Buildah is output log-rusage on
the server side of podman-remote.  It is using os.Stdout rather
then options.Out by default.

[NO TESTS NEEDED] This can only be tested on the Podman side,
existing tests should show if this breaks anything.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@dcermak PTAL
@containers/podman-maintainers PTAL

File: imagebuildah/executor.go [hunk 1]
```diff
@@ -184,15 +184,11 @@ func NewExecutor(logger *logrus.Logger, store storage.Store, options define.Buil
...
-		if options.RusageLogFile == "" {
-			rusageLogFile = os.Stdout
-		} else {
-			rusageLogFile, err = os.OpenFile(options.RusageLogFile, os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
-			if err != nil {
-				return nil, err
-			}
+	rusageLogFile := options.Out
+	if options.LogRusage && !options.Quiet && options.RusageLogFile != "" {
+		rusageLogFile, err = os.OpenFile(options.RusageLogFile, os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
```
Comment (nalind) [inline imagebuildah/executor.go:189]: When does this file get closed?

**Review by ashley-cui — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
/lgtm
/hold
# until green tests

**Pull request merged**